pkgs/development/python-modules/dataclass-csv/default.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  distutils,
+  setuptools,
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  pname = "dataclass-csv";
+  version = "1.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dfurtado";
+    repo = pname;
```

```suggestion
    repo = "dataclass-csv";
```
using pname like this is an antipattern because it can break overriding